R/ceb.R

```diff
@@ -0,0 +1,126 @@
+#' @title Calculate mean children ever born & living
+#'
+#' @description Use default arguments to calculate mean children ever born
+#' related statistics from a DHS individual recode file.
+#'
+#' @param data \[`data.frame()`\]\cr
+#'   Individual recode dataset.
+#' @param by \[`formula()`\]\cr
+#'   A formula specifying factor variables by which to stratify analysis.
+#'   Default is 'NULL' to specify no stratification. Passed to [`survey::svyby()`].
+#' @param agegr \[`numeric()`\]\cr
+#'   The start of each age group (in years) to group each individual's age by
+#'   and to stratify analysis by. Can be 'NULL' to specify no age-groups should
+#'   be created and stratified by. Default is 15, 20, 25, 30, 35, 40, 45, 50.
+#' @param clusters \[`formula()`\]\cr
+#'   A formula specifying cluster ids to be passed as `ids` to survey::svydesign.
+#'   Must correspond to variables in `data`. Default is '~v021'.
+#' @param strata \[`formula()`\]\cr
+#'   A formula specifying strata to be passed as `strata` to survey::svydesign.
+#'   Must correspond to variables in `data`. Default is '~v024+v025'.
+#' @param weight \[`character(1)`\]\cr
+#'   Variable name for sample weight. Default is 'v005'.
+#' @param ceb \[`character()`\]\cr
+#'   Variable names for children ever born related columns to calculate the mean
+#'   & standard error of. Value corresponds to name in `data`, name corresponds
+#'   to new variable name in output. Default is 'c(ceb = "v201", lc = "v218")'.
+#' @param dob \[`character(1)`\]\cr
+#'   Variable name for date of birth of each individual. Default is 'v008'.
+#'   Required when `agegr` is not 'NULL'.
+#' @param intv \[`character(1)`\]\cr
+#'   Variable name for interview date for each individual. Default is 'v008'.
```

Comment:
Does `each individual` mean `each child` or `each mother`? I think `individual` should be replaced to be more clear here.

Reply:
Changed these to refer to the `interviewee` since the documentation seems to imply questions about children ever born can be asked of women (page 49) & men (page 137) in the survey https://dhsprogram.com/pubs/pdf/DHSG4/Recode7_DHS_10Sep2018_DHSG4.pdf.

I agree especially for the fertility and child mortality functions in `demogsurv` there could be clearer documentation on if the variable is referring to the mother or child. These functions assume lots of familiarity with household surveys which I think is fair.